torchci/lib/drciUtils.ts

```diff
@@ -553,3 +564,16 @@ export async function isSameAuthor(
   // * Cherry picking
   return isSameEmail || isSameCommitUsername || isSamePrUsername;
 }
+
+export function isRunnerDisconnectionFailure(job: RecentWorkflowsData): boolean {
+  const runnerDisconnectionPatterns = [
+    /The runner has received a shutdown signal/i,
+    /The operation was canceled/i,
```

Comment:
Jobs can be cancelled for other reasons and I think it still ends up saying this
```suggestion
```